inception-project/inception
# #2466 - Displayed Text changes as annotations are added / removed

**What's in the PR**
- Drop reliance on tokens for rendering data in brat - instead use whitespace in the text to generate UI-only tokens
- This is a "backport" of a set of changes previously being introduced as part of #1778 Make tokenization editable

**How to test manually**
* Import a document that does not have tokens on all characters - e.g. not on dashes and underscores as described in the issue description
* See if it now renders properly irrespective of the presence of a wide annotation

**Automatic testing**

**Documentation**
